.rubocop.yml

```diff
@@ -18,6 +18,9 @@ AllCops:
 Bundler/DuplicatedGem:
   Enabled: false
 
+Gemspec/DevelopmentDependencies:
```

Comment:
See discussion at https://instructure.slack.com/archives/C02G4TMN1JQ/p1742314108281659; tl;dr we've made the decision that we _do_ want use this cop. it's a pretty easy change to comply